### ISSUE TITLE
Efficient Neural Mapping for Localisation of Unmanned Ground Vehicles. (arXiv:2211.04718v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#23534]
Efficient Neural Mapping for Localisation of Unmanned Ground Vehicles. (arXiv:2211.04718v1 [cs.RO])<br>
https://ift.tt/xRr2p7G<br>
<p>Global localisation from visual data is a challenging problem applicable to many robotics domains. Prior works have shown that neural networks can be trained to map images of an environment to absolute camera pose within that environment, learning an implicit neural mapping in the process. In this work we evaluate the applicability of such an approach to real-world robotics scenarios, demonstrating that by constraining the problem to 2-dimensions and significantly increasing the quantity of training data, a compact model capable of real-time inference on embedded platforms can be used to achieve localisation accuracy of several centimetres. We deploy our trained model onboard a UGV platform, demonstrating its effectiveness in a waypoint navigation task. Along with this work we will release a novel localisation dataset comprising simulated and real environments, each with training samples numbering in the tens of thousands.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
